src/Utilities/Compiler/Extensions/IOperationExtensions.cs

```diff
@@ -410,6 +410,24 @@ public static IOperation GetRoot(this IOperation operation)
             return operation;
         }
 
+        /// <summary>
+        /// Sorta like <see cref="GetRoot(IOperation)"/>, but not the root root.
+        /// </summary>
+        /// <param name="operation"></param>
+        /// <returns></returns>
+        public static IOperation GetInnermostFunction(this IOperation operation)
+        {
+            while (operation.Parent != null
+                && operation.Kind != OperationKind.AnonymousFunction
+                && operation.Kind != OperationKind.LocalFunction)
+            // What's a FlowAnonymousFunction?
```

Comment:
> What's a FlowAnonymousFunction?

IAnonymousFunctionOperation exists in high level operation tree, which is rewritten into IFlowAnonymousFunctionOperation by the control flow graph builder.